modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -191,6 +211,69 @@ public void testGetOpenAPI() throws Exception {
 			categorizationDisabledObjectDefinition);
 	}
 
+	@Test
+	public void testGetOpenAPIFilterableFields() throws Exception {
+		ObjectDefinition objectDefinition =
+			ObjectDefinitionLocalServiceUtil.getObjectDefinition(
+				_objectDefinition.getObjectDefinitionId());
+
+		ObjectDefinition relatedObjectDefinition =
+			ObjectDefinitionTestUtil.publishObjectDefinition(
+				"Object5",
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING, true, true, null,
+						"field", "field", false)),
+				ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		ObjectRelationshipLocalServiceUtil.addObjectRelationship(
+			null, TestPropsValues.getUserId(),
+			_objectDefinition.getObjectDefinitionId(),
+			relatedObjectDefinition.getObjectDefinitionId(), 0,
+			ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+			LocalizedMapUtil.getLocalizedMap("relationship1"), "relationship1",
+			false, ObjectRelationshipConstants.TYPE_MANY_TO_MANY, null);
+		ObjectRelationshipLocalServiceUtil.addObjectRelationship(
+			null, TestPropsValues.getUserId(),
+			_objectDefinition.getObjectDefinitionId(),
+			relatedObjectDefinition.getObjectDefinitionId(), 0,
+			ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+			LocalizedMapUtil.getLocalizedMap("relationship2"), "relationship2",
+			false, ObjectRelationshipConstants.TYPE_ONE_TO_MANY, null);
+
+		Set<String> expectedFilterableFields = new HashSet<>(
+			_getObjectDefinitionFilterableFields(objectDefinition));
```

Comment:
Having a method that calculates the expected filterable fields does not make sense! We know which fields are the ones that must appear in the list of filterable. We are attaching our test to the way we developed the solution

Reply:
> We know which fields are the ones that must appear in the list of filterable.
yeah these are the one that are part of entity fields map. how else do you want to know which fields we need to verify?